aries-site/src/pages/components/index.js

```diff
@@ -1,33 +1,59 @@
 import React from 'react';
-import { Layout, NavPage } from '../../layouts';
-import { DescriptiveHeader, Meta, ComingSoon } from '../../components';
-import { getPageDetails } from '../../utils';
+import Link from 'next/link';
+import { Box, Heading, Paragraph } from 'grommet';
+
+import { CardGrid, ContentCard, Meta } from '../../components';
+import { structure } from '../../data';
+import { Layout, PageIntro } from '../../layouts';
+import { getPageDetails, getParentPage, nameToPath } from '../../utils';
 
 const title = 'Components';
-const page = getPageDetails(title);
+const pageDetails = getPageDetails(title);
 
-const Components = () => {
-  const descriptiveHeader = (
-    <DescriptiveHeader
-      background={page.color}
-      subText={page.description}
-      icon={page.icon}
-      title={title}
-    />
-  );
+const cards = structure
```

Comment:
Can we move this to utils? maybe it could be called `getCards`

Reply:
Yep. Good call and already in progress. Using the extra time to DRY this PR out.

Reply:
@taysea Implemented.